### PR TITLE
Use Node.js to run script

### PR DESCRIPTION
**Pull request by cafca**
- [x] Create NPM package (package.json)
- [x] Add .gitignore
- [x] Adapt `dumpjson.html` code into `main.js` and `converter.js`, make converter a module with exports
- [x] Write output to file system instead of `document`
- [x] Run original Wahl-o-Mat JS statements in sandbox
- [x] Create boilerplate to parametrize main script for different output formats:
- [x] Export of `occasions`
- [x] Export of `occasiondata `
- [x] Export of `parties`
- [ ] Export of `theses_categories`
- [ ] Export of `categories`
~- [ ] Adapt `alltheses.html`~~
~- [ ] Adapt `alltheses_categories.html`~
~- [ ] Adapt `alltheses_direct.html`~
~- [ ] Adapt `dumpcsv.html`~
~- [ ] Adapt `metadata.html`~
- [x] Add docstrings 
- [x] Add instructions for setup and execution of script

To run:
1. Install node.js, npm
2. Run `npm install` in root
3. Run `node src/main.js`

### CODE REVIEW

File: src/options.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+},
+{
+  name: 'verbose',
+  description: 'Enable verbose output',
+  alias: 'v',
+  type: Boolean
+},
+{
+  name: 'csv',
+  description: 'Export all occastions and theses as csv',
```
Comment (friesenkiwi) [inline src/options.js]: => occasions

File: src/options.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  type: Boolean
+},
+{
+  name: 'target',
+  alias: 't',
+  description: 'Target directory for saving files. Default is `../export/`.',
```
Comment (friesenkiwi) [inline src/options.js:47]: won't work, if target directory (e.g. default export) is not already created

File: src/converter.js [hunk 3]
```diff
@@ -46,6 +66,76 @@ function load_data_by_occasionfile(occasionFile, finalFunction) {
   });
 }
 
+exports.wahlomat_convert_theses = function(collectedData) {
```
Comment (friesenkiwi) [inline src/converter.js]: original functions with changed behaviour are already present below, probably missed when merging

**Review by friesenkiwi — CHANGES_REQUESTED** [review]
OK, dann sollten wir am besten die write_ und load_-Methoden in eine andere Vanilla-JS-Datei auslagern, so dass die von alltheses, alltheses_categories, metadata und parties zum Darstellen der Daten aus dem JSON genutzt werden können. die _direct.html und die dump*.html können dann komplett weg.

File: src/options.js (hunk 2 above)
Comment (cafca) [reply src/options.js:47]: fixed

File: src/options.js (hunk 1 above)
Comment (cafca) [reply src/options.js]: fixed

File: src/converter.js (hunk 3 above)
Comment (cafca) [reply src/converter.js]: fixed

**Review by friesenkiwi — APPROVED** [review]

**Pull request merged**